src/event/event-type.js

```diff
@@ -201,7 +201,11 @@ const CustomEventType: PKEventTypes = {
   /**
    * Fired when the vr stereo mode changed
    */
-  VR_STEREO_MODE_CHANGED: 'vrstereomodechanged'
+  VR_STEREO_MODE_CHANGED: 'vrstereomodechanged',
+  /**
+   * Fired when too many frames are dropped
+   */
+  EXCEEDED_MAX_FRAME_DROP: 'exceededmaxframedrop'
```

Comment:
This is an internal event and not a public player event. you shouldn't expose it here but do define it in the quality watcher class.